tor-relay/README.md

```diff
@@ -9,3 +9,4 @@
 | **CONTACT_EMAIL**            | Your contact email                                                           | none          |
 | **RELAY_BANDWIDTH_RATE**     | Limit how much traffic will be allowed through your relay (must be > 20KB/s) | 100 KBytes    |
 | **RELAY_BANDWIDTH_BURST**    | Allow temporary bursts up to a certain amount                                | 200 KBytes    |
+| **RELAY_PORT**               | Default port to used for incoming Tor connections (ORPort)                   | 9001          |
```

Comment:
Typo? I think this should read "Default port used for incoming Tor connections (ORPort)"